scripts/update_teams.py

```diff
@@ -53,7 +53,14 @@ def create_team(org, name, description, repos):
         org.url + "/teams",
         input=post_parameters
     )
-    return github.Team.Team(org._requester, headers, data, completed=True)
+    team = github.Team.Team(org._requester, headers, data, completed=True)
+
+    for repo in repos:
+        # Ensure that we have merge rights on the repo for this team.
+        url = team.url + "/repos/" + repo._identity
+        team._requester.requestJsonAndCheck("PUT", url, input={"permission": "push"})
```

Comment:
As `repo_names` is already passed to the team, we can then just drop this `for`-loop entirely after that change.